### DISCUSSION TITLE
Bug Bounty

### DISCUSSION THREAD

**Discussion by MarziehHashemi** [mate-desktop/engrampa#505] [Show and tell]
I found an Engrampa vulnerability from the MATE desktop environment. Does MATE Desktop Environment pay a bounty to get it? Also, this vulnerability has a CVEID from mitre. This vulnerability has not yet been published anywhere.

**Comment by lukefromdc** [comment]
No posted bounties from the team have ever been issied that I know of.

**Comment by raveit65** [comment]
Mate is a noncommercial project.

**Comment by lukefromdc** [comment]
Look on Bountysource for any bounties posted by outside parties against bugs in MATE. I collected a few back in 2017 before Bountysource went Paypal-only and I was unable to cash out any more bounties.
